runtime/runc_jailer.go

```diff
@@ -465,6 +465,12 @@ func (j *runcJailer) bindMountFileToJail(src, dst string) error {
 // exposeBlockDeviceToJail will call mknod on the block device to ensure
 // visibility of the device
 func exposeBlockDeviceToJail(dst string, rdev, uid, gid int) error {
+	if _, err := os.Stat(dst); err == nil {
```

Comment:
Why do we have the conflicting file?

Reply:
I don't know where the file is being created. I plan to work on this later today, to track it down. If you have some ideas what are some places I should look into would be very helpful.

Reply:
I may not be understanding the issue so I have a question:

Is this function creating a file different from the one created by EBS/EC2 when attaching the volume? For example, assume `/dev/nvme5n1` as the device file for an EBS Volume attached to EC2. If we are passing that same `/dev/nvme5n1` to this function, then it will definitely cause a conflict because it has already been created on our behalf when EBS volume was attached.

Am I understanding the workflow correctly?

Reply:
This function will create a device file in a different directory. The dst passed into the function is diffrent from /dev/nvm5n1, more precisely, it will look like 
```
/var/lib/firecracker-containerd/shim-base/default#test-drive-mount/rootfs/dev/nvme4n1
```
default is namespace, test-drive-mount is VMID.